nnbuilder-api/src/main/proto/tasksqueue.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package org.hse.nnbuilder.services;
+
+enum TaskName {
+  LearnNN = 0;
+  ApplyToData = 1;
+}
+
+enum TaskStatus {
+  HaveNotStarted = 0;
+  Processing = 1;
+  Done = 2;
+}
+
+service TasksQueueService {
+  rpc createtask (TaskCreationRequest) returns (TaskCreationResponse);
+  rpc getinformation(GettingInformationRequest) returns (GettingInformationResponse);
+}
+
+// Creating new task
+message TaskCreationRequest {
+  TaskName name = 1;
+  int64 nnId = 2;
+  int64 dataId = 3;
+}
+
+message TaskCreationResponse {
+  int64 taskId = 1;
+}
+
+// Asking information about a task
+message GettingInformationRequest {
+  int64 taskId = 1;
+}
+
+message GettingInformationResponse {
+  TaskName taskName = 1;
+  int64 nnId = 2;
+  int64 dataId = 3;
+  string startTime = 4;
```

Comment:
There's a built-in timestamp type

Reply:
U mean 
<img width="195" alt="image" src="https://user-images.githubusercontent.com/70476032/168851815-2a10eb70-ff73-4f29-9cfb-b2f9fae89ffc.png">

Reply:
https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Timestamp

Reply:
You need to import it first and then use with fully qualified name, I think